integration-tests/deployment/ccip/deploy.go

```diff
@@ -475,8 +475,10 @@ func DeployChainContracts(
 
 	tx, err := rmnRemote.Contract.SetConfig(chain.DeployerKey, rmn_remote.RMNRemoteConfig{
 		RmnHomeContractConfigDigest: activeDigest,
-		Signers:                     []rmn_remote.RMNRemoteSigner{},
-		MinSigners:                  0, // TODO: update when we have signers
+		Signers: []rmn_remote.RMNRemoteSigner{
+			{NodeIndex: 0, OnchainPublicKey: common.Address{1}},
```

Comment:
Maybe better to leave this empty, this is gonna get filled out in another PR that is working on RMN integration

Reply:
I have to add at least one signer, even if it's a bogus one, in order to fulfill the `n >= 2f+1` requirement on the contract